### DISCUSSION TITLE
Resto v0.1.3

### DISCUSSION THREAD

**Discussion by abdfnx** [abdfnx/resto#12] [Releases]
## What's Changed
* add `run` command to run restofiles. by @abdfnx 
   ![demo](https://user-images.githubusercontent.com/64256993/147034869-4d1120fa-9611-4f55-84e5-4071791ce692.gif)
* update `ci` workflow @david-tomson 
* upgrade `release` workflow by @abdfnx 

## Changelog

* 2c4e3eb add `docker` instructions, add resto shortcuts in `README.md`
* 001bd78 update the code owners (@abdfnx, @david-tomson, @Timothee-Cardoso)
* 6f527c9 update `Install from GitHub CLI as extension` step name in `ci` workflow (#11)
* 12843ad add `-f` flag to fix `Run Restofile` step in `ci` workflow
* 98a78cc add `go` job in `ci` workflow, update test scripts, make `from_gh_cli` needs go
* 737dc38 Bump github.com/briandowns/spinner from 1.17.0 to 1.18.0 (#10) @dependabot
* 323ef29 restore `Push changes` step in `release` workflow for `gh-resto` repo
* b68d566 and again 🙄: update `Commit files` step in `release` workflow
* 820429b again 🙄: update `Commit files` step in `release` workflow for `gh-resto`
* de2e3b3 update `Commit files` step in `release` workflow for `abdfnx/gh-resto`
* 016cd32 remove `tmp` from git ignore list
* 6e71ac5 fix `release` workflow yaml syntax
* 9d88040 update `gh-rs.js` script, use `ad-m/github-push-action` action to push changes
* 844ddbc `echo ${{ secrets.ACCESS_TOKEN }} | gh auth login --with-token`
* e31e073 update `git config` commands in `release` workflow
* 4d11e33 add `ref: ${{ github.head_ref }}` arg to `actions/checkout@v2` action
* 97101fd add `git config --global` commands to config author in `release` ci
* fea1c16 fix `Build gh-resto` step by add `cd ../scripts/gh-resto` command
* 389d97c add setup `node.js` with version `16` step for `gh-resto` step
* 14cdb42 add `yarn` command to install dependencies for `gh-resto`
* bb91518 create script to update `gh-resto` version and date, update `release` ci
* 19e6660 remove output
* 1fd2223 run `goreleaser release` manually in `Build` step: `release` workflow
* ec393a3 😑 trying to fix `release` workflow by give all permissions to workflow
* 43507ad add `id` for `Set up Date` step to use output for goreleaser env vars
* 590f363 update `build_resto` job name, use goreleaser action to build
* 219120f add new permissions to `release` workflow, update github token
* 1883c58 add `ACTIONS_ALLOW_UNSECURE_COMMANDS` env var to `ci` workflow
* cd4a25f make `from_gh_cli` job needs `bfs` & `bfs_windows` to run
* 53cfb33 add `echo "::add-path::C:UsersrunneradminAppDataLocalrestobin;"` command to fix
* 210587f create windows build from source script file, update scripts & `ci` workflow
* 0110d3c fix `release` workflow by fixing `secrets` typo
* 5ab60ad setup the security policy of resto `.github/SECURITY.md`
* de9c157 add `Ctrl+Q` shortcut for bodyeditor of `run` command, add flags docs
* eda5fb6 add flags docs for `install` and `run` commands in README
* 2fcdec1 Merge branch 'dev' of https://github.com/abdfnx/resto into dev
* 569c792 finish from support `Restofile` #5, add `resto run` command, update all
* 1c203cf add `AUTHOR_EMAIL` env var secret to goreleaser build step
* b6c2268 determine every os with arch, and fix all goreleaser issues
* cc3ce10 remove `freebsd` os, add `.apk` nfpms linux format in goreleaser config file
* cb8486b remove `openbsd` os, add `freebsd` build to the formatted archives in goreleaser
* 3722f10 update goreleaser oss and arches
* 3e21dd4 support new oss and arches, add checksum template name
* fb6e09d put `Email` env var in Quotations marks in goreleaser config file
* b201956 add `homebrew` configuration in goreleaser config file
* 5c3c56e add some information about `resto install` command in README

**Full Changelog**: https://github.com/abdfnx/resto/compare/v0.1.2...v0.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/abdfnx/resto/releases/tag/v0.1.3'>Resto v0.1.3</a>.</em>

**Comment by david-tomson** [comment]
where's `freebsd` ?

**Reply by abdfnx** [reply]
yeah I see, because I forgot to add `freebsd` to archives builds

```yml
# .goreleaser.yml
- id: nix
    builds: [ macos, linux ]
```

so I'm going to add it
